spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -212,7 +212,7 @@ The `@EnableBinding` annotation can take as parameters one or more interface cla
 
 [NOTE]
 ====
-In Spring Cloud Stream 1.0, the only supported bindable components are the Spring Messaging `MessageChannel` and its extensions `SubscribableChannel` and `PollableChannel`.
+In Spring Cloud Stream 1.0, the only supported bindable components are the Spring Messaging `MessageChannel` and its extension `SubscribableChannel`.
 Future versions should extend this support to other types of components, using the same mechanism.
```

Comment:
Given we have `KStream` components ready to be merged, we can refer it here. We can track it via https://github.com/spring-cloud/spring-cloud-stream/issues/1019